src/main/java/org/myrobotlab/service/RasPi.java

```diff
@@ -482,7 +487,7 @@ public boolean getWiringPi() {
           pinList.add(pindef);
         }
       } else {
-        log.error("Unknown boardtype {}", SystemInfo.getBoardType());
+        log.error(String.format("Unknown boardtype %s", SystemInfo.getBoardType()));
       }
     } catch (UnsupportedOperationException | IOException | InterruptedException e) {
       // TODO Auto-generated catch block

```

Comment:
same logging thing .. if you see these - please replace with 
log.error("some intelligent text", e);  <- this will have some time text and still dump the stack trace